ansible_3par_docker_plugin/properties/plugin_configuration_properties_sample.yml

```diff
@@ -9,6 +9,7 @@ INVENTORY:
     hpe3par_ip: 192.168.1.50
     hpe3par_username: 3paradm
     hpe3par_password: 3pardata
+    hpe3par_port: "8080"
```

Comment:
@bhagyashree-sarawate Please maintain the same syntax for adding properties in this file.
Please do `hpe3par_port: 8080` instead.

Reply:
done